.ci/Jenkinsfile

```diff
@@ -400,6 +400,12 @@ def pyrun(command){
             label: "Installing dependencies"
           )
         }
+        sh(script: """#!/bin/bash
+          python --version
+          ${env.WORKSPACE}/.venv/bin/${command} --version
```

Comment:
But command will contain the command and the args. See https://github.com/elastic/e2e-testing/pull/2568/commits/98e929df3138c4cb57ccf3427bdd95cf12c34923#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR409

```
${command.split(' ')[0]}
```